platforms/core-runtime/versioned-cache/src/main/java/org/gradle/internal/versionedcache/CacheVersionMapping.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2018 the original author or authors.
+ * Copyright 2024 the original author or authors.
```

Comment:
>I am always confused whether these should get updated or not 🤔

I've reverted these changes, but it would be great to have a good answer for that. If only IDEA wasn't so persistent in trying to change these...